### PR TITLE
Added file with skipped conformance tests

### PR DESCRIPTION
**Pull request by AlexMuresan**

### CODE REVIEW

**Comment by test-e2e-win** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: 
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **adelina-t**

Assign the PR to them by writing `/assign @adelina-t` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/e2e-win/e2e-win-prow-deployment/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by test-e2e-win** [comment]
Hi @AlexMuresan. Thanks for your PR.

I'm waiting for a [e2e-win](https://github.com/orgs/e2e-win/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/e2e-win/people) to skip this step.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by test-e2e-win** [comment]
Hi @AlexMuresan. Thanks for your PR.

I'm waiting for a [e2e-win](https://github.com/orgs/e2e-win/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/e2e-win/people) to skip this step.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: exclude_conformance_test.txt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+[sig-storage] Projected  should be consumable from pods in volume with mappings and Item mode set [NodeConformance] [Conformance]
+[sig-storage] Projected  should be consumable from pods in volume with mappings and Item Mode set [NodeConformance] [Conformance]
+[sig-storage] Projected  should be consumable from pods in volume with mappings as non-root [NodeConformance] [Conformance]
+[sig-storage] Projected  should set DefaultMode on files [NodeConformance] [Conformance]
+[sig-storage] Projected  should set mode on item file [NodeConformance] [Conformance]
+
+[sig-storage] Downward API volume  should set DefaultMode on files [NodeConformance] [Conformance]
+[sig-storage] Downward API volume  should set mode on item file [NodeConformance] [Conformance]
+
+"[sig-storage] EmptyDir volumes  should support (non-root,0644,default) [NodeConformance] [Conformance]"
```
Comment (adelina-t) [inline exclude_conformance_test.txt]: These quotes don't make it very easy for copy/paste. Nor parsing.

Comment (AlexMuresan) [reply exclude_conformance_test.txt]: Removed them. Should I remove the spaces as well?

Comment (adelina-t) [reply exclude_conformance_test.txt]: Neah, spaces are fine

**Comment by test-e2e-win** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/e2e-win/e2e-win-prow-deployment/pull/32#issuecomment-407402687" title="Approved">adelina-t</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/e2e-win/e2e-win-prow-deployment/blob/master/OWNERS)~~ [adelina-t]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**